src/constants_json_strings.py

```diff
@@ -138,7 +143,20 @@
 OUTPUT_BUS_NAME = "output_bus_name"
 INPUT_BUS_NAME = "input_bus_name"
 ENERGY_VECTOR = "energyVector"
-
+EXCESS = " excess"
+BUS_SUFFIX = " bus"
+
+# DSO
+DSO_CONSUMPTION = "_consumption"
+DSO_FEEDIN = "_feedin"
+DSO_CONSUMPTION_PERIODS = "_consumption_period_"
+CONNECTED_CONSUMPTION_SOURCES = "connected_consumption_sources"
+CONNECTED_FEEDIN_SINK = "connected_feedin_sink"
+
+# Autogenerated assets
+AUTO_SOURCE = "_source"
+AUTO_SINK = "_sink"
+DISPATCHABILITY = "is_dispatchable"
```

Comment:
```suggestion
DISPATCHABILITY = "dispatchable"
```

this should solve the failing tests of D1

Reply:
@SabineHaas niiiice! Thanks!